multiapps-controller-api/pom.xml

```diff
@@ -72,6 +68,12 @@
             <artifactId>jaxb-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>1.6.16</version>
+            <scope>compile</scope>
```

Comment:
- Why this dependency is needed now?
- All dependencies versions are set in parent pom.xml, not in child pom.xml.
- Is it really necessary to add this dependency at compile time? Isn't it necessary only at build time?